scrap.js

```diff
@@ -17,19 +19,82 @@ async function fetchLeaders(id) {
   return leaders
 }
 
-async function fetchOrgsWithLeaders() {
+async function searchGitHubOrgs(query) {
+  const token = process.env.GH_TOKEN
```

Comment:
[`.travis.yml`](https://github.com/coala/gci-leaders/blob/master/.travis.yml#L10) uses `GITHUB_TOKEN` var.
So ideally this should use the same var.

Why do we even need this? API rate limit?

Reply:
Yea, the API does not like us making a bunch of requests at once without a token. I'll change it to `GITHUB_TOKEN` now!